interview-questions.md

```diff
@@ -10,40 +10,70 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 Your answer:
 
+Object oriented programming is a programming model that helps break down large programming projects, which makes it best for collaborating. Also known as OOP, it allows programmers the ability to store data that can be changed or updated without affecting the whole code. OOP is object based meaning it can contains instances of classes.
+
 Researched answer:
 
+Object oriented programming is a programming model that was created to offset the difficulty programmers were having with working on large software systems. Programmers use OOP to create containers for data that can be manipulated wihtout affecting the entire program. With OOP, programmers can section and delegate specific areas of the code to specific programmers to work on individually making it the most popular and widely used programming model. It relies on the concept of classes and object, meaning it uses reusable pieces of code called classes, to then create individual instances of objects. Within these classes there are functions, referred to a methods in OOP, that can only be used with objects of that type. The functions are defined within the class, then perform some action on the object. All of these features make OOP faster and easier to execute by providing a clear readable and reusable structure for the program allowing the code to be easier to modify, debug, and maintain. 
```

Comment:
Great job digging into what OOP is.  Make sure to also compare the differences from functional programming as well.